.github/workflows/gh-ci.yaml

```diff
@@ -56,11 +56,12 @@ jobs:
 
     - name: install package
       run: |
-        python -m pip install -v .
+        python -m pip install -v ./panedrlite
+        # python -m pip install -v ./panedr
```

Comment:
we should add an extra step to check imports and make sure stuff works.

Something like this would do the trick (not tested the working-directory line so 🤞🏽 )

```
- name: test imports
  # Exit the git repo in order for pbr to stop auto-picking up version info from the local git data
  working-directory: ../
  run: |
    python -Ic "from panedrlite import edr_to_dict"
    python -lc "from panedr import edr_to_df"
```

Reply:
I have done this now, I guess we will only see on merge if the import test works?